projects/kit/components/avatar/avatar.component.ts

```diff
@@ -59,8 +59,12 @@ export class TuiAvatarComponent {
         return this.avatarUrl !== null && this.isUrlValid;
     }
 
+    get iconAvatar(): boolean {
+        return !!this.avatarUrl?.startsWith(`tuiIcon`);
```

Comment:
Would this need an extra check here if you were using icons via `svgService.define` ?